app/javascript/components/buyers/BuyerSnapshot.jsx

```diff
@@ -0,0 +1,32 @@
+import PropTypes from "prop-types";
+import React from "react";
+import Touchable from 'rc-touchable';
+
+class BuyerSnapshot extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+  
+  navigateToBuyer = () => {
+    window.location = `/buyers/${this.props.buyer.id}`;
+  }
+
+  render() {
+    return (
+      <Touchable onPress={() => this.navigateToBuyer()}>
```

Comment:
Hmm, I think it would be more intuitive for only the name and avatar clickable. Having the entire div as a `Touchable` makes it difficult to copy the email, too

Reply:
that is very true, thanks!

Reply:
adding a hover state would probably be nice, but i think we can hold off until button refactoring to do that